src/psyclone/tests/psyir/backend/sympy_writer_test.py

```diff
@@ -47,14 +48,29 @@
 from psyclone.psyir.symbols import BOOLEAN_TYPE, CHARACTER_TYPE
 
 
-def test_sym_writer_constructor():
+def test_sym_writer_constructor(monkeypatch):
     '''Test that the constructor accepts an optional dictionary.
     '''
     sympy_writer = SymPyWriter({'some': 'symbol'})
     assert sympy_writer._sympy_type_map['some'] == 'symbol'
     # Also test that not specifying a type map as argument works:
     sympy_writer = SymPyWriter()
     assert sympy_writer._sympy_type_map == {}
+    assert sympy_writer._DISABLE_LOWERING is True
+
+    def error(_):
```

Comment:
I think from hereon should be a new test - maybe test_lowering_disabled or something.

Reply:
Done